compiler/natives/src/crypto/sha256/sha256.go

```diff
@@ -0,0 +1,67 @@
+// +build js
+
+package sha256
+
+import (
+	"errors"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+var ErrWebCryptoFailed = errors.New("crypto: Web Crypto API call failed")
```

Comment:
Can you elaborate on how it will be used?

I'm concerned about this because I don't think we should be adding to the API surface of `crypto/...` packages in standard library, as that would make user code incompatible with Go.